backend/restapi/services/recommendation.py

```diff
@@ -0,0 +1,45 @@
+from openai import OpenAI
+import os
+
+
+def get_recommendations(phrases, tags):
```

Comment:
Refactor to `GptRecomendationService` class and create private function to call chat gpt API with selected model